examples/post-item-with-pystac.ipynb

```diff
@@ -0,0 +1,290 @@
+    "    return s3_client.get_object(Bucket=bucket, Key=key)[\"Body\"].read()\n",
+    "\n",
+    "STAC_IO.read_text_method = read_text_s3\n",
+    "\n",
+    "item = Item.from_file(\"s3://rasterfoundry-development-data-us-east-1/berlin-catalog/image/berlin-sentinel.json\")\n",
+    "(item, item.get_parent())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Find collections in Franklin\n",
+    "\n",
+    "Franklin advertises its collections under the `/collections` endpoint. For this example we'll use an existing collection called `berlin` to POST items to."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<Collection id=berlin>]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "\n",
+    "[Collection.from_dict(x) for x in requests.get(\"http://localhost:9090/collections\").json()[\"collections\"]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding items to Franklin\n",
+    "\n",
+    "Franklin accepts POSTs to `/collections/<id>/items` with STAC 0.9.0 items. Since we have one of those, we can add it to the `berlin` collection:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<Item id=image>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "url = \"http://localhost:9090/collections/berlin/items\"\n",
+    "resp = requests.post(url, json=item.to_dict())\n",
+    "result_item = Item.from_dict(resp.json())\n",
+    "result_item"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'type': 'Feature',\n",
+       " 'stac_version': '0.9.0',\n",
+       " 'id': 'image',\n",
+       " 'properties': {'datetime': '2018-03-03T10:20:19Z'},\n",
+       " 'geometry': {'type': 'Polygon',\n",
+       "  'coordinates': [[[13.742925226720015, 52.64467528345354],\n",
+       "    [13.330610521659846, 52.63948385599755],\n",
+       "    [13.340993376571825, 52.363530645448044],\n",
+       "    [13.756192207996433, 52.36860670784946],\n",
+       "    [13.742925226720015, 52.64467528345354]]]},\n",
+       " 'bbox': [13.330610521659846,\n",
+       "  52.363530645448044,\n",
+       "  13.756192207996433,\n",
+       "  52.64467528345354],\n",
+       " 'links': [{'rel': 'parent',\n",
+       "   'href': 'http://localhost:9090/api/collections/berlin',\n",
+       "   'type': 'application/json',\n",
+       "   'label:assets': []},\n",
+       "  {'rel': 'root',\n",
+       "   'href': '../catalog.json',\n",
```

Comment:
I think it might make sense to clean the item up if possible before we post it. For instance, the root would be `<parameterized-host>` -- and the link to the collection can come from the collection we retrieved above.

Reply:
I'm not sure this is going to be super doable -- what even is the root catalog, since STAC API doesn't do catalogs? Or are you suggesting that the root should be the api root?

Reply:
Yeah - the root is the API root for the API I believe -- there are only collections that belong to the root catalog.